src/VecSim/index_factories/tiered_factory.cpp

```diff
@@ -132,6 +237,10 @@ size_t EstimateInitialSize(const TieredIndexParams *params) {
         est += TieredHNSWFactory::EstimateInitialSize(params);
         bf_params = TieredHNSWFactory::NewBFParams(params);
     }
+    if (params->primaryIndexParams->algo == VecSimAlgo_SVS) {
+        est += TieredSVSFactory::EstimateInitialSize(params);
+        bf_params = TieredSVSFactory::NewBFParams(params);
+    }
 
     est += BruteForceFactory::EstimateInitialSize(&bf_params, false);
     return est;
```

Comment:
why do we return the size of the bf index if HAVE_SVS=0?

Reply:
TBH, tried to minimize code change here.
Unfortunately, I did not understand from existing code, which value to be returned if `params->primaryIndexParams->algo` value is not supported (e.g. `VecSimAlgo_BF`).
Please, just let me know, what is expected - I will fix the logic.